docs/metrics/ordo.md

```diff
@@ -32,25 +32,26 @@
 
 | Metric | Value |
 | ------ | ----- |
-| AnnotationAssertion | 170039 |
+| AnnotationAssertion | 204442 |
 | SubAnnotationPropertyOf | 1 |
-| Declaration | 15174 |
+| Declaration | 15177 |
 | SubClassOf | 43059 |
 
 
 #### Entity namespaces: axiom counts by namespace
 
 | Metric | Value |
 | ------ | ----- |
-| dc11 | 1 |
 | prefix_unknown | 15157 |
 | oboInOwl | 6 |
 | owl | 2 |
-| rdf | 1 |
-| IAO | 1 |
 | xsd | 2 |
+| skos | 3 |
```

Comment:
here's one place where the change in the mapping predicate (using the 3 skos predicates now) is shown